src/types/onyx/Policy.ts

```diff
@@ -169,6 +169,19 @@ type UberReceiptPartner = {
      * organization id for connected uber
      */
     organizationID?: string;
+
+    /**
+     * Mapping of workspace member email to Uber employee status
+     */
+    employees?: Record<
```

Comment:
This seems wrong, no? The uber employee data is under `receiptPartners.uber.employees`